extensions/wikia/CookieSyncer/templates/CookieSyncer_Index.php

```diff
@@ -1 +1,9 @@
-<iframe src="<?= $url; ?>" class="cookie-syncer-module-iframe"></iframe>
+<script type="text/javascript">
+	if (window.document.cookie.indexOf('cookiesync_done') === -1 && window.document.cookie.indexOf('tracking-opt-in-status') !== -1) {
+		var iframe = document.createElement('iframe');
+		iframe.style.display = "none";
+		iframe.src = "<?= $url; ?>";
+		iframe.classList.add("cookie-syncer-module-iframe");
```

Comment:
We can skip either `iframe.style.display = "none";` or setting this class.

Reply:
@rogatty fix in 5259cdd8dc2